### PR TITLE
updated cas

### PR DESCRIPTION
**Pull request by matthewl20**

### CODE REVIEW

**Comment by matthewl20** [comment]
I updated Travis yml to install Elasticsearch 1.7.3 instead of using Elasticsearch 5 which was causing our tests to fail.  Let me know if there is a better way to do this.  I based it off of:  https://docs.travis-ci.com/user/database-setup/#ElasticSearch

**Comment by matthewl20** [comment]
I also updated rails to 4.2.

File: lib/cas_authentication.rb [hunk 1]
```diff
@@ -1,43 +1,47 @@
...
+    eduPersonEntitlement: { entitlements: Array },
+    eduPersonAffiliation: { affiliations: Array }
+  }.freeze
 
   def present?
     session['cas'].present?
   end
 
   def new_user?
-    !!user.try(:new_record?)
```
Comment (rift137) [inline lib/cas_authentication.rb]: why unboolean this?

Comment (matthewl20) [reply lib/cas_authentication.rb]: It's a Rubyocop Offence and `user.try(:new_record?)` always returns true or false

Comment (matthewl20) [reply lib/cas_authentication.rb]: Fixed.

**Pull request merged**